act-rules-format.bs

```diff
@@ -202,13 +202,13 @@ Rules are tested in two steps. Firstly, the applicability is used to find a list
 
 This will mean that every time a rule is executed on a page, it will return a set with zero or more results, each of which MUST have at least the following properties:
 
-- Rule Identifier (test)
-- Test Subject (Web page)
-- Test Target (pointer)
-- Outcome (`Passed`, `Failed`, or `Inapplicable`)
+- [Rule Identifier](#unique-identifier) (test)
+- [Test Subject](#output-test-subject) (Web page)
+- [Test Target](#output-test-target) (pointer)
+- [Outcome](#output-outcome) (`Passed`, `Failed`, or `Inapplicable`)
 
 <div class=example>
-Output data using EARL and JSON-LD
+Output data using EARL and JSON-LD. (See [Evaluation and Report Language (EARL) 1.0](https://www.w3.org/TR/EARL10-Schema/) [[EARL10-Schema]] and [Java Script Object Notation (JSON)](https://www.json.org).) 
```

Comment:
I suggest using this link to the overview page for EARL: https://www.w3.org/WAI/intro/earl

Reply:
Done.